### PR TITLE
docs: add media chapter

### PR DESCRIPTION
**Pull request by Tandashi**
## **What happens in this Pull Request**
In this Pull Request, a new chapter explaining the different types of Media in Motion Canvas has been added.
This includes `Audio`, `Image` and `Video`.

Furthermore, two new examples have been added (`media-image` and `media-video`).
The assets for those examples can be located in the newly created `assets` directory.

Lastly, the `base` path for the build examples has been passed to `Vite`.
Without this, assets of examples would not be able to load when embedded into the documentation.
A more in-depth explanation can be located inside commit 6d85f8728dd9e59f1d7fde5f51b9f4fb8ef0beac.

---

Closes: #209

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

File: packages/docs/docs/getting-started/media.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+# Media
+
+In this chapter we will take a look at all the different types of media Motion
+Canvas lets you add to your animations.
+
+## Audio
+
+One type of media Motion Canvas supports is audio. To add an audio track to your
+animation you would need to edit your `project.ts` file. First we need to import
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:16]: nit: "would" is in a little bit of weird tense here. "will," or omitting the word entirely is a bit more straightforward:
> To add an audio track to your animation, you need to edit your `project.ts` file.

File: packages/docs/docs/getting-started/media.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+```ts
+// my-animation/
+// ├─ audio/
+// │  └─ voice.mp3
+// ├─ src/
+// ...
+import audio from '../audio/voice.mp3';
+```
+
+One imported we now need to let Motion Canvas know that we want to use this
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:28]: "Once imported, we need to let Motion Canvas know that we want to use this audio file." (One -> Once, remove "now")

File: packages/docs/docs/getting-started/media.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+[`ProjectConfiguration`](/api/core/index/ProjectConfig):
+
+```ts
+export default makeProject({
+  // ...
+  audio: audio,
+});
+```
+
+If we open the editor now we should see that your audio track has been added.
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:40]: "If we open the editor now, we should see that your audio track has been added" (added a comma)

File: packages/docs/docs/getting-started/media.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+![](@site/static/img/media/editor-audio-track.png)
+
+However sometimes its not ideal to start with the audio at the beginning but
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:44]: "However, it's not always ideal to start with the audio at the beginning. Sometimes it's better to start with a bit of delay."

File: packages/docs/docs/getting-started/media.mdx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+### Supported Formats
+
+A list of supported file formats can be found
+[here](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Audio_codecs#common_codecs)
+
+## Images
+
+Another type of media Motion Canvas supports are images. To use an image in the
+animation we first need to import the image:
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:67]: "To use an image in the animation, we need to import the image:"

File: packages/docs/docs/getting-started/media.mdx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+```ts
+// my-animation/
+// ├─ images/
+// │  └─ example.png
+// ├─ src/
+// ...
+import examplePng from '../../images/example.png';
+```
+
+Then we can use Motion Canvas [`Image`](/api/2d/components/Image) component to
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:78]: nit: "as follows" and colon tend to have the same purpose. I'd omit "as follows" to keep it more straightforward.

File: packages/docs/docs/getting-started/media.mdx [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+import examplePng from '../../images/example.png';
+
+export default makeScene2D(function* (view) {
+  view.add(<Image src={examplePng} />);
+});
+```
+
+Images can also be animated like every other component in Motion Canvas. For
+this we just need to create a [`Reference`](/docs/references) and pass that to
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:93]: "For this, we just need to..."

File: packages/docs/docs/getting-started/media.mdx [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+<AnimationPlayer name="media-image" small />
+
+### Supported Formats
+
+A list of supported file formats can be found
+[here](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types)
+
+## Video
+
+Videos in Motion Canvas work similar to images. Firstly we need to import our
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:122]: nit: "First," is a bit more concise than "Firstly," (and either way should have a comma)

File: packages/docs/docs/getting-started/media.mdx [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+export default makeScene2D(function* (view) {
+  const videoRef = createRef<Video>();
+
+  view.add(<Video ref={videoRef} src={exampleMp4} />);
+});
+```
+
+Now that we have a [`Reference`](/docs/references) to the
+[`Video`](/api/2d/components/Video) element we can use it to animate the video
```
Comment (hhenrichsen) [inline packages/docs/docs/getting-started/media.mdx:163]: nit: "...to the Video element, we can use it to animate the video, as well as play and pause it." (added some commas)

**Review by hhenrichsen — COMMENTED** [review]
None of these are blockers for me, like usual, but there are a couple things that make the content a little more clear / concise.

**Comment by aarthificial** [comment]
Thanks for the PR!
A note for the future: you used the phrase "Motion Canvas" 7 times in this one doc. I'd try to limit this to one or two mentions per page.

**Pull request merged**